server/DatabaseConnection.js

```diff
@@ -0,0 +1,33 @@
+import pg from 'pg';
+
+require('dotenv').config();
+let pool;
+console.log('env', process.env.NODE_ENV);
```

Comment:
I think we have talked about this before. Always remove `console.log` before pushing.